snakedwi/workflow/rules/diffusion/eddy.smk

```diff
@@ -251,7 +251,7 @@ if config["use_eddy_gpu"]:
                     if value == True
                 ]
             ),
-            container=config["singularity"]["fsl_gpu"],
+            container=config["singularity"]["fsl_abspath"],
```

Comment:
Is there a reason you directly call singularity in the command here? It's completely possible to pass the `--nv` to singularity via `--singularity-args`, and then we can put the actual docker URL in config rather than a domain-specific path

Reply:
One reason was that if `--nv` gets used for all the singularity calls, it spits out a warning if there is no GPU on the node it is running on.. Unless you have a way of setting singularity-args on a rule-specific basis? I am also ok with the warning as it does make it cleaner. 

But the other reason was for the bedpost cpu-based parallelization script - I recall I used singularity exec there because the FSL container doesn't have GNU parallel in it..

Reply:
Hmm, unfortunately I don't think there's a way to do that.

I've had `--nv` in my profile for quite a while and don't really recall seeing any warnings... I guess they're not too prominent?

The bedpost thing makes sense. Out of curiousity, what does the GNU parallel command do that a bunch of `command&; command&; ...` calls doesn't? More efficient?

Reply:
Ah, parallel will act like a scheduler, running the jobs in parallel with the specified number of cores in use. Eg if you have 200 commands and 8 cores it will run the commands to keep the 8 cores busy until they are all done..

Reply:
But I am fine with moving to the docker uri for everything else and just telling people to use --nv in singularity-args.. Bedpost is optional in any case, and at some point could also make a container that has parallel in it as well for that..